bedita-app/models/behaviors/cacheable.php

```diff
@@ -199,6 +199,10 @@ public function clearCache(&$model, $objectId, $reset = true, $descendants = tru
             $this->addObjectsToClean($model, $this->getObjectsToCleanById($model, $objectId));
         }
 
+        if (!$this->on) {
```

Comment:
Why not at the beginning of the method?

Reply:
I thought that maybe something may be relying on the `$this->addObjectsToClean()` call… but maybe not. 🤔 

I'll move it at the beginning.

Reply:
Maybe yes, but I think it's more clear there.